### ISSUE TITLE
Use more reasonable defaults

### ISSUE THREAD

**Issue by z00m1n** [geerlingguy/ansible-role-homebrew#47]
At the moment, the default settings in [`main.yml`](https://github.com/geerlingguy/ansible-role-homebrew/blob/master/defaults/main.yml) install `ssh-copy-id` and `pv` packages, `homebrew/core` and `caskroom/cask` taps and the `firefox` cask.

These defaults are arbitrary at best; `main.yml` seems to me like 'active' sample code with mostly documentary purposes. The user is required to take extra action to undo those most likely unwanted settings - and for that, they need to find the file and get the syntax right.

Instead, I propose to not install any arbitrary components by default, but document how this is achieved and link to official documentation.

Pull request in the making.

**Comment by elliotweiser** [comment]
I agree with the reasoning here. The defaults should be empty and the usage should be documented 👍 

I think the original PR was closed since @z00m1n's fork was deleted. Will look into addressing this myself.

**Comment by josephholsten** [comment]
@elliotweiser @z00m1n anything else we need in this PR?

**Comment by elliotweiser** [comment]
@josephholsten Just looked at the commit in which you mentioned this issue. I think that was the essence of the original PR i.e. _don't install any packages/casks unless the user says otherwise_. Go ahead and open the PR and I'll review.

**Comment by josephholsten** [comment]
wow, how did I PR myself? opening against the real repo...

**Comment by elliotweiser** [comment]
@josephholsten One thing I forgot to mention-- the defaults documented in the README.md will need to be updated as well.
